autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers() abort
-    return ['go', 'golint', 'govet']
+    return ['go', 'golint', 'govet', 'gometalinter']
```

Comment:
I don't think it should be preferred, it just runs those same tools.

My other change, is to set golint errors as informational, as they are stylistic. Separating golint from gometalinter makes this check straightforward.

It's certainly possible to combine them, but I don't really see a lot of the value, especially if you want to support both.

It maybe possible to drop golint and combine that into gometalinter, but I'm not see a lot of the value (other than potentially removing `neomake#makers#ft#go#golint`).

As go vet is available with all Go commands, if a user does not have gometalinter installed, they at least get go vet out of the box.